src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_taskqueue.v1beta2.json.baseline

```diff
@@ -7,39 +7,31 @@
 // 2. Install the C# client library by adding a dependency on the relevant NuGet
 //    package. Libraries published by Google are owned by google-apis-packages:
 //    https://www.nuget.org/profiles/google-apis-packages
-
 using Google.Apis.Auth.OAuth2;
 using Google.Apis.Services;
 using Google.Apis.Taskqueue.v1beta2;
 using System;
-using System.Threading.Tasks;
 
 using Data = Google.Apis.Taskqueue.v1beta2.Data;
```

Comment:
Is this ordering/spacing for `using Data ...` intentional, or just an artifact of assembly?